packages/horizon/ignition/modules/periphery/EpochManager.ts

```diff
@@ -0,0 +1,19 @@
+import { buildModule } from '@nomicfoundation/hardhat-ignition/modules'
+import { deployWithGraphProxy } from '../proxy/GraphProxy'
+
+import ControllerModule from './Controller'
+import EpochManagerArtifact from '@graphprotocol/contracts/build/contracts/contracts/epochs/EpochManager.sol/EpochManager.json'
```

Comment:
I noticed these imports in a few files. Are these npm packages that you had already published? Are these contracts verified already? I'm mostly asking to understand:

1. Are you loading from `node_module` or from this monorepo? If the latter, why?
2. Should we do anything with its build info/sources so that then it can be verified easily?

Reply:
Good question. So, Horizon project (`packages/horizon`) relies on some contracts from the current version of the protocol (`packages/contracts`) which remain mostly unchanged. In order to be able to use modern tooling and make it easier for auditors we decided to spin up a new package (`horizon`) where we put all the new stuff then eventually the old stuff that's required will be moved to `horizon` and `contracts` will be deprecated.

So to answer your questions, anything we are pulling from `@graphprotocol/contracts` is being pulled from the monorepo package `contracts` and those are already deployed to production and verified. I don't know what would happen if I tried to verify something from the `contracts` package from within `horizon` 🤔 but in our specific case that will never happen as anything we pull from there should already be verified.

Reply:
Ok, I think we are good for now. But it's an interesting use case. I think we may want to let you provide their build infos. Do you keep track of them?

Reply:
Not sure I follow! Keep track of what?